spec/components/layout_for_public_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe "Layout for public", type: :view do
+  def component_name
+    "layout_for_public"
+  end
+
+  it "adds a default <title> tag" do
+    render_component({})
+
+    assert_select "title", visible: false, text: "GOV.UK - The best place to find government services and information"
+  end
+
+  it "adds a custom <title> tag" do
+    render_component(title: "Custom GOV.UK Title")
+
+    assert_select "title", visible: false, text: "Custom GOV.UK Title"
+  end
+
+  it "can support full width layouts" do
```

Comment:
**Reviewing commit by commit**

Does this need a corresponding test to check non-full width layouts?